executor/aggregate.go

```diff
@@ -722,7 +722,7 @@ func (e *HashAggExec) parallelExec(ctx context.Context, chk *chunk.Chunk) error
 func (e *HashAggExec) unparallelExec(ctx context.Context, chk *chunk.Chunk) error {
 	// In this stage we consider all data from src as a single group.
 	if !e.prepared {
-		err := e.execute(ctx)
+		err := e.execute(ctx, chk.RequiredRows())
```

Comment:
chunk size has a limit, 1024. If I run a sql like `desc analyze select distinct a from t limit 3000;` , the chk.RequiredRows() will be 1024 and it will get a wrong result for the sql.
`